src/JuliusSweetland.OptiKey.Core/JuliusSweetland.OptiKey.csproj

```diff
@@ -286,7 +286,9 @@
     <Compile Include="Models\KeyboardOutputServiceState.cs" />
     <Compile Include="Models\KeyValueConverter.cs" />
     <Compile Include="Models\PluginMetadata.cs" />
+    <Compile Include="Models\XmlKeyboardModels\XmlDynamicItem.cs" />
     <Compile Include="Models\XmlKeyboardModels\XmlDynamicKey.cs" />
+    <Compile Include="Models\XmlKeyboardModels\XmlMultiStepKey.cs" />
```

Comment:
I changed the name of my original XmlDynamicKey to XmlMultiStepKey so that I could define XmlDynamicKey as a new class for my new keyboard code.